.travis.yml

```diff
@@ -17,8 +17,8 @@ script:
 
 - docker build --target test --tag my-test-image .
 - docker run my-test-image tests/test_view_model.py
-- docker run my-test-image tests/test_integration.py
-- docker run -e TRELLO_API_KEY -e TRELLO_API_TOKEN my-test-image tests/test_selenium.py
+# - docker run my-test-image tests/test_integration.py
+# - docker run -e TRELLO_API_KEY -e TRELLO_API_TOKEN my-test-image tests/test_selenium.py
```

Comment:
It would be good to get at least your end-to-end tests running
* We'll need to use the new environment variables for Mongo instead of the Trello variables
* We'll want the test to set a new database name, and to clear that db at the end
  * Otherwise we should be able to get rid of the set up code